src/bootstrap/src/bin/main.rs

```diff
@@ -147,6 +148,10 @@ fn main() {
             t!(file.write_all(lines.join("\n").as_bytes()));
         }
     }
+
+    if env::var("BOOTSTRAP_PROFILE").is_ok_and(|v| v == "1") {
```

Comment:
We already check this environment variable elsewhere. Could you please introduce a function so that the logic is shared? Thanks.